oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -923,10 +935,20 @@ impl Runtime {
         label: &Label,
         initial_handle: oak_abi::Handle,
     ) -> Result<(), OakStatus> {
-        info!("creating node with config: {:?}", config);
+        info!(
+            "creating Node with config: {:?} and label: {:?}",
+            config, label
+        );
```

Comment:
As above, already `debug`-logged in `RuntimeProxy`

Reply:
Done.